packages/byods/src/BYODS.ts

```diff
@@ -1,47 +1,389 @@
 /* eslint-disable no-console */
-import fetch from 'node-fetch';
-import {LocalStream} from '@webex/media-helpers'; // Just to show that you can import other packages from the workspace
+import fetch, {RequestInit, Response} from 'node-fetch';
+import {jwksCache, createRemoteJWKSet} from 'jose';
+
+interface ApiResponse<T> {
+  data: T;
+  status: number;
+}
 
 interface SDKConfig {
   clientId: string;
   clientSecret: string;
+  baseUrl: string;
+}
+
+const userAgent = 'BYoDS NodeJS SDK / 0.0.1';
```

Comment:
Create a constants file and keep these there

Reply:
Done